### DISCUSSION TITLE
Domain routing like in next.config i18n

### DISCUSSION THREAD

**Discussion by sigurdarson** [vinissimus/next-translate#991] [Q&A]
next `12.2`
next-translate `1.5.0`

From the documentation I assumed I could use any properties utilised in Next i18n, I might have been wrong.

> "In the configuration file you can use both the configuration that we [specified here](https://github.com/aralroca/next-translate#3-configuration) and the own [features about internationalization of Next.js 10](https://nextjs.org/docs/advanced-features/i18n-routing)."

I've been trying to get "Domains" to work but do not seem to be able. Maybe that's not possible with this package, not sure.

This is test code, what I'm trying to achieve is to use domain routing for locales, `example.se` displays `se` content instead of routing to `example.co/se`. The code does build properly and 3/4 routes work properly and in my case the `se` route does not work, it just redirects to `example.co / default locale`.

App is being hosted on Google cloud run, and both domains have been routed to the service.

`i18n.json`
```
{
  "locales": ["en", "da", "no", "se"],
  "defaultLocale": "en",
  "domains": [
    {
      "domain": "staging.example.co",
      "defaultLocale": "en"
    },
    {
      "domain": "staging.example.se",
      "defaultLocale": "se"
    }
  ],
  "pages": {
    "*": ["index", "common"]
  }
}
```

Any ideas if I might be doing something wrong or maybe it's not possible in the current version. any help is greatly appreciated.

**Comment by aralroca** [comment]
Your `i18n.json` looks fine, and it should work, we pass domains to the next.config and we are using domains in some projects, and works fine...

https://github.com/aralroca/next-translate-plugin/blob/275da3e43fc925da3571e74a81849b51e7d876fb/src/index.ts#L46

I think is an issue no related to next-translate, more about google cloud configuration or another thing... I convert this issue to discussion and I hope someone can help you about this.
